public/js/event-app.js

```diff
@@ -120,25 +120,34 @@ $(document).ready(function() {
         var firstScriptTag = document.getElementsByTagName('script')[0];
         firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
 
+        this.currentUser = new models.User(USER);
+        this.currentUserIsAdmin = function() {
+          return this.currentUser.isAdminOf(curEvent);
```

Comment:
No, because this.currentUserIsAdmin() is only called in the context of the initializer. I've tested it, it works. :) Did you have some other approach in mind?